libs/sql-schema-describer/src/mysql.rs

```diff
@@ -392,6 +401,16 @@ impl<'a> SqlSchemaDescriber<'a> {
         Ok(map)
     }
 
+    fn db_generated_date_time(default_string: &str, default_generated: bool) -> DefaultValue {
+        if default_generated {
```

Comment:
I am wondering whether the boolean parameter wouldn't be better as a function call (so only handle literals in this function). But really a nitpick.